test/acceptance/tests.py

```diff
@@ -544,12 +544,18 @@ def test_student_info_no_submission(self):
         Scenario: staff tools indicates if no submission has been received for a given learner
 
         Given I am viewing the staff area of an ORA problem
+        And I myself have submitted a response with self-assessment
         When I search for a learner in staff tools
         And the learner has not submitted a response to the ORA problem
         Then I see a message indicating that the learner has not submitted a response
         And there are no student information sections displayed
         """
         self.auto_auth_page.visit()
+
+        # This is to catch a bug that existed when the user viewing staff tools had submitted an assessment,
+        # and had a grade stored (TNL-4060).
+        self.do_self_assessment()
```

Comment:
I tried to write a unit test for this instead, but it was very difficult because it involves creating a submission for the user requesting the learner information (not too hard), and grading that submission (hard). This was an easy change to make in an existing bok choy test, and I verified it fails without the bug fix.